src/Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -188,10 +187,10 @@ private function legacyConstructor($args)
      *
      * @return array
      */
-    private function getIdentifierValues($entity)
+    private function getNormalizedIdentifier($entity)
```

Comment:
Maybe this could be removed

Reply:
if the same exception is thrown in `modelManager->getNormalizedIdentifier` then ye